website2/docs/topology-development-streamlet-api.md

```diff
@@ -113,34 +113,26 @@ $ heron submit local \
 
 ### Java Streamlet API starter project
 
-If you'd like to up and running quickly with the Heron Streamlet API for Java, you can clone [this repository](https://github.com/streamlio/heron-java-streamlet-api-example), which includes an example topology built using the Streamlet API as well as the necessary Maven configuration. To build a JAR with dependencies of this example topology:
-
-```bash
-$ git clone https://github.com/streamlio/heron-java-streamlet-api-example
-$ cd heron-java-streamlet-api-example
-$ mvn assembly:assembly
-$ ls target/*.jar
-target/heron-java-streamlet-api-example-latest-jar-with-dependencies.jar
-target/heron-java-streamlet-api-example-latest.jar
-```
+If you'd like to up and running quickly with the Heron Streamlet API for Java, you can view the example topologies [here](https://github.com/apache/incubator-heron/tree/master/examples/src/java/org/apache/heron/examples/streamlet)
```

Comment:
Can the link here use something like `.../tree/{{ heron:version }}/...` so that baked versions have the correct link?

Reply:
addressed.